sdk/core/Azure.Core/api/Azure.Core.net5.0.cs

```diff
@@ -398,6 +400,18 @@ public abstract partial class ClientOptions
         public static bool operator !=(Azure.Core.ContentType left, Azure.Core.ContentType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public abstract partial class DelayStrategy
```

Comment:
I considered using Delay rather than DelayStrategy but I felt it becomes too confusing when passing in TimeSpan delay values and in docs. I'm open to reconsidering, but left as DelayStrategy for now.

Reply:
/cc @tg-msft @m-nash